apps/webstore/src/app/shared/components/items/checkout-item/checkout-item.component.html

```diff
@@ -21,9 +21,9 @@
             >
         </div>
         <div class="info-container">
-            <div class="info-line">
+            <!-- <div class="info-line">
```

Comment:
@Teodor25 If you don't want it to show, remove it completely

Why do you want to remove it though?